components/org.wso2.carbon.identity.api.server.application.management/org.wso2.carbon.identity.api.server.application.management.v1/src/main/resources/applications.yaml

```diff
@@ -2578,6 +2578,14 @@ components:
           type: boolean
           example: false
           description: Decides whether user consent needs to be skipped during logout flows.
+        useExternalConsentManagement:
+          type: boolean
+          example: false
```

Comment:
Can we set a default value to false

Reply:
I have set the default value as false for this.

Reply:
we can set a value in swagger as well right?

Reply:
Resolved by https://github.com/wso2/identity-api-server/pull/434/commits/eee60c5adeaf9335a30d63b7f64010aa3d601cf6